### PR TITLE
Add Austrian furniture stores

### PR DESCRIPTION
**Pull request by hikemaniac**
This PR adds the following Austrian furniture stores:

- Kika ([Q256456](https://www.wikidata.org/wiki/Q256456))
- Leiner ([Q1661472](https://www.wikidata.org/wiki/Q1661472))
- Mömax (a subsidiary of _Möbelix_)
- Rutar ([Q22910157](https://www.wikidata.org/wiki/Q22910157))

### CODE REVIEW

File: brands/shop/furniture.json [hunk 1]
```diff
@@ -354,6 +374,14 @@
       "shop": "furniture"
     }
   },
+  "shop/furniture|Mömax": {
```
Comment (tas50) [inline brands/shop/furniture.json]: Without a wikidata value this one won't actually show up in editors.

Comment (hikemaniac) [reply brands/shop/furniture.json]: Okay, thank you. The problem is, that (like _XXXLutz_) it belongs to the company "Möbelix". It has in fact got two Wikidata pages - one individual one with barely any data and a redirect to "Möbelix" (in the same way as _XXXLutz_).

**Comment by hikemaniac** [comment]
I've added the missing Wikidata items, everything should be working now. 😁

**Comment by bhousel** [comment]
> I've added the missing Wikidata items, everything should be working now.

Great, thanks for the quick additions! 👍

**Pull request merged**